.github/workflows/verify_library_generation.yaml

```diff
@@ -5,14 +5,13 @@ on:
   pull_request:
     paths:
     - library_generation/**
-
   workflow_dispatch:
 name: verify_library_generation
 jobs:
   integration_tests:
     strategy:
       matrix:
-        java: [ 8 ]
+        java: [ 11 ]
```

Comment:
I think the generator still supports Java 8 runtime, why increase it to 11?

Reply:
This was part of the setup to run the showcase script. It is not necessary anymore - removing